### PR TITLE
iOS Simulator stdout

### PR DESCRIPTION
**Pull request by jwerle**

### CODE REVIEW

**Review by chicoxyzzy — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
Note: this one includes commits from #441

**Comment by chicoxyzzy** [comment]
Windows build fails here

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
let's also remove this line in `ci.yml` on the `ios` job

```yaml
if: ${{ false }}
```

**Comment by mribbons** [comment]
Awesome :+1:

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -174,7 +174,6 @@ jobs:
 
     - name: test a socket api example app
       # remove this line if you want to test the Android build on macOS
```
Comment (chicoxyzzy) [inline .github/workflows/ci.yml:176]: This comment is not necessary. It seems that it's also incorrect

**Pull request merged**